lib/dao.js

```diff
@@ -1532,20 +1532,23 @@ function NumberType(val) {
  * @returns {Object} The coerced where clause
  * @private
  */
-DataAccessObject._coerce = function(where) {
+DataAccessObject._coerce = function(where, props) {
```

Comment:
Please rename `props` to `propertyDefinitions` and add a jsdoc entry describing this new parameter.